.travis.yml

```diff
@@ -17,4 +21,4 @@ before_install:
 script:
   - ./ci/scripts/build_projects.sh
   - ./ci/scripts/cppcheck.sh
-  - ./ci/scripts/astyle.sh
+  - if [[ -n "$ASTYLE_FORMAT" ]]; then ./ci/scripts/astyle.sh; fi
```

Comment:
you could move this check inside the `./ci/scripts/astyle.sh` script

Reply:
I guess in this case, a wrapper script would look nicer.
One that calls all 3 scripts based on the variable.

One example is:
* https://github.com/analogdevicesinc/linux/blob/master/.travis.yml#L13
* https://github.com/analogdevicesinc/linux/blob/master/ci/travis/run-build.sh#L33